packages/main/src/themes/RatingIndicator.css

```diff
@@ -54,6 +54,7 @@
 	position: relative;
 	width: var(--_ui5_rating_indicator_item_width);
 	height: var(--_ui5_rating_indicator_item_height);
+	font-size: max(var(--_ui5_rating_indicator_min_font_size), 100%);
```

Comment:
with this code you restrict the app to give custom sizes to the rating indicator, which is not the task - the task is to make sure that when no custom sizing is given out component has a certain size in order to fulfil the requirement of the min touch area. The little size of the component in compact is due to the height and width of the property being set to 0.67em, nothing to do with the font-size

Reply:
Ok, I will remove this restriction and leave the applications to decide on custom sizes.